userspace/libsinsp/test/sinsp_with_test_input.cpp

```diff
@@ -255,10 +255,18 @@ sinsp_evt* sinsp_with_test_input::generate_proc_exit_event(int64_t tid_to_remove
 
 sinsp_evt* sinsp_with_test_input::generate_random_event(int64_t tid_caller)
 {
-	// Generate a random event on init to trigger the removal after proc exit
+	// Generate a random failed event. Useful when we want to trigger some actions on an event
+	// and we don't care about the event chosen.
```

Comment:
While you are at it could you change this function name, such as `generate_getcwd_failed_entry_event()`? Fortunately the event is not random